src/lang/uk.rs

```diff
@@ -399,162 +393,271 @@ const KOPIYKAS: [[&str; 6]; 2] = [
+            ),
+            Currency::EUR => format!(
+                "євроцент{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::GBP => format!(
+                "пенс{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::IDR => format!(
+                "сен{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::INR => format!(
+                "пайс{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::ILS => format!(
+                "агор{}",
+                NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::JPY => format!(
+                "сен{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::KRW => format!(
+                "чон{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::KZT => format!(
+                "тиїн{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::MYR => format!(
+                "сен{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
             Currency::NOK => String::from("оре"),
             Currency::PEN => String::from("сентімо"),
-            Currency::PLN => format!("грош{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::QAR | 
-            Currency::RIYAL |
-            Currency::SAR => format!("філс{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::RUB => //format!("копійк{}", NOUN_1ST_GROUP_FEMININE_HARD_DECLENSIONS[number_idx][declenation_idx]),
-                             String::from("руській воєнний корабль іді нахуй"),
-            Currency::THB => format!("cатанг{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::TRY => format!("лір{}",NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::UAH => String::from(KOPIYKAS[number_idx][declenation_idx]),
+            Currency::PLN => format!(
+                "грош{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::QAR | Currency::RIYAL | Currency::SAR => format!(
+                "філс{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::RUB =>
+            //format!("копійк{}", NOUN_1ST_GROUP_FEMININE_HARD_DECLENSIONS[number_idx][declination_idx]),
+            {
+                String::from("руській воєнний корабль іді нахуй")
```

Comment:
I'm torn between keeping this or putting the original `format!` just to make sure the lib is usable haha